pkg/cloud/scope/mocks/ec2_mock.go

```diff
@@ -0,0 +1,525 @@
+/*
+Copyright The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Code generated by MockGen. DO NOT EDIT.
+// Source: ../ec2.go
```

Comment:
Where is this new mock being used?

Reply:
Initially thought to create all mock files in scope folder, but later found it should be within each service package.
So I forgot to cleanup this, but now updated the PR. Thanks